src/ui/map.js

```diff
@@ -407,6 +407,22 @@ class Map extends Camera {
     }
 
     /**
+     * Gets the map's geographical bounds.
+     *
+     * Returns the LngLatBounds by which pan and zoom operations on the map are constrained.
+     *
+     * @returns {LngLatBounds | null} The maximum bounds the map is constrained to, or `null` if none set.
+     */
+    getMaxBounds () {
+        if (!(!Array.isArray(this.transform.latRange) || this.transform.latRange.length === 0) &&
```

Comment:
I find this a little hard to read  😕 can we invert the check ( `if (this.transform.latRange && this.transform.latRange.length == 2) && ... else { return null; } `) for clarity?